docs/test-playwright.md

```diff
@@ -0,0 +1,95 @@
+# Playwright test
+
+This workflow executes Playwright-based tests in a controlled and isolated environment via GitHub Actions.
+
+The workflow can:
+
+- execute a building step, both for node and PHP environments (if the PHP version is provided and a `composer.json` file is present)
+- create an environment variables file named `.env.ci` dedicated to the test step; load this file using `dotenv-ci` directly in your test script, e.g., `./node_modules/.bin/dotenv -e .env.ci -- npm run e2e`
+- execute the tests using Playwright
+- upload the artifacts
+
+**Simplest possible example:**
+
+```yml
+name: E2E Testing
+
+on:
+  workflow_dispatch:
+jobs:
+  e2e-playwright:
+    uses: inpsyde/reusable-workflows/.github/workflows/test-playwright.yml@main
+    with:
+      ARTIFACT_PATH: './artifacts'
+      SCRIPT_NAME: 'ci-test-e2e'
+```
+
+## Configuration parameters
+
+### Inputs
+
+| Name                            | Default                         | Description                                                                                       |
+|---------------------------------|---------------------------------|---------------------------------------------------------------------------------------------------|
+| `ARTIFACT_INCLUDE_HIDDEN_FILES` | `false`                         | Whether to include hidden files in the artifact                                                   |
+| `ARTIFACT_NAME`                 | `'artifact'`                    | Name for the artifact                                                                             |
+| `ARTIFACT_OVERWRITE`            | `false`                         | Determine if an artifact with a matching name will be deleted before a new one is uploaded or not |
+| `ARTIFACT_PATH`                 |                                 | A file, directory or wildcard pattern that describes what to upload                               |
+| `ARTIFACT_RETENTION_DAYS`       | `30`                            | Duration after which artifact will expire in day                                                  |
+| `COMPOSER_DEPS_INSTALL`         | `false`                         | Whether to install Composer dependencies.                                                         |
```

Comment:
I know we need to improve how we document (#53), but the convention is not to have full stops in the table. 🙈
```suggestion
| `COMPOSER_DEPS_INSTALL`         | `false`                         | Whether to install Composer dependencies                                                          |
```